.dockerignore

```diff
@@ -4,3 +4,8 @@
 **/Dockerfile
 **/*.md
 **/*.Rproj
+output_dir/**/*
+output_dir
+pacta-data/**/*
+pacta-data
```

Comment:
Do you need both of these ignores here?

Reply:
I think so. I was getting odd behaviors (long build times) when I was trying to build without both.